tests/migration/eviction_strategy.go

```diff
@@ -109,6 +107,7 @@ var _ = SIGMigrationDescribe("Live Migration", func() {
 				Expect(resVMI.Status.EvacuationNodeName).To(Equal(""), "vmi evacuation state should be clean")
 			})
 
+			// Does it really makes sense?
```

Comment:
Please develop a little bit!

Reply:
Each iteration waits for the pdb deletion before the next one.
So, basically, there is no correlation between each iteration.
But this was a note for me, and I forgot to delete :)